tests/base.py

```diff
@@ -25,7 +25,10 @@ class SnapchatBase(unittest.TestCase):
     INCREMENTAL = "INCREMENTAL"
     OBEYS_START_DATE = "obey-start-date"
 
+    # TDL-21946 Work-Item to generate test data for following list of streams
     stats_streams = {"ad_account_stats_daily", "ad_account_stats_hourly", "pixel_domain_stats", "campaign_stats_daily", "campaign_stats_hourly", "ad_squad_stats_daily", "ad_squad_stats_hourly", "ad_stats_daily",  "ad_stats_hourly"}
+    # Currently there is no data available for following targeting streams
+    missing_targeting_streams = {"targeting_interests_dlxp", "targeting_interests_plc"}
```

Comment:
Any specific reason we have kept declared 2 sets here?

Alternately, we can keep one set instead of 2 and logically separate them since for those are anyways missing streams from the test point of view.

```suggestion
    missing_targeting_streams = {
    # TDL-21946 Work-Item to generate test data for following list of streams
    "ad_account_stats_daily", "ad_account_stats_hourly", "pixel_domain_stats", "campaign_stats_daily", "...", "ad_stats_hourly",
    
    # Currently there is no data available for following targeting streams
    "targeting_interests_dlxp", "targeting_interests_plc"}
```

Reply:
These are two separate groups of streams(one is stats related and another is interests related groups), they have their own schema and own pagination values.
combining these two would require more changes in pagination test and keeping them separate would help us track down them as groups for those data is missing.